howto/update/upgrade-anbox.md

```diff
@@ -106,10 +106,10 @@ If you don't run any of the services in a high availability configuration, upgra
 
```

Comment:
@morphis We need to update the upgrade topic for Appliance as well but since you are already working on it, I didn't include this as part of the release PR.